extension/content/components/common/AppSidebar.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { Icon, Nav, Sidenav, Popover, Whisper } from "rsuite";
+
+import { useEnvironmentState } from "devtools/contexts/environment";
+import { splitCamelCase } from "devtools/utils/helpers";
+
+export default function AppSidebar(): JSX.Element {
+  const { selectedKey } = useEnvironmentState();
+
+  const versionPopover = (
+    <Popover>
+      <dl>
+        {Object.entries(__BUILD__).map(([key, value]) => {
+          return (
+            <>
+              <dt>{splitCamelCase(key, { case: "title-case" })}</dt>
+              <dd>
+                {typeof value === "string" ? value : JSON.stringify(value)}
+              </dd>
+            </>
+          );
+        })}
+      </dl>
+    </Popover>
+  );
+
+  return (
+    <div className="app-sidebar">
+      <Sidenav className="h-100">
+        <Sidenav.Body className="d-flex flex-column h-100">
+          <Nav vertical className="flex-grow-1">
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="book" />}
+              to={`/${selectedKey}/recipes`}
+            >
+              Recipes
+            </Nav.Item>
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="filter" />}
+              to={`/${selectedKey}/filters`}
+            >
+              Filters
+            </Nav.Item>
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="table" />}
+              to={`/${selectedKey}/pref-studies`}
+            >
+              Pref Studies
+            </Nav.Item>
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="puzzle-piece" />}
+              to={`/${selectedKey}/addon-studies`}
+            >
+              Add-on Studies
+            </Nav.Item>
+          </Nav>
+
+          <div className="version-info flex-grow-0">
```

Comment:
a couple of these will only land in the details page patch but...

you could use:
`text-subtle cursor-pointer p-2 flex-grow-0`

Reply:
I'll use all of these, even though some of them don't work. When your details patch comes through it will get better.